tests/test_extras_mm_fields.py

```diff
@@ -459,37 +490,46 @@ class MyBadObj:
             def __init__(self, some_field: int) -> None:
                 self.some_field = some_field
 
-        class MyMmSchema(marshmallow.Schema):
+        class DumpMyMmSchema(marshmallow.Schema):
             class Meta:
                 strict = False
 
             periodo_tributario = RcvPeriodoTributarioField(
                 required=True,
-                load_from='source field name',
             )
             other_field = marshmallow.fields.Integer(
                 required=False,
             )
 
-        class MyMmSchemaStrict(marshmallow.Schema):
+        class LoadMyMmSchema(marshmallow.Schema):
             class Meta:
-                strict = True
+                strict = False
```

Comment:
> Setting the `strict` option on `class Meta` has no effect on **`Schema`** behavior.

Source: https://marshmallow.readthedocs.io/en/stable/upgrading.html#schemas-are-always-strict

⚠️ **Check all files** for cases like this. The following command will be useful for that:

```sh
grep -rE 'strict = (True|False)'
```

```suggestion
```